src/protocol/ipa_prf/boolean_ops/addition_sequential.rs

```diff
@@ -258,9 +269,8 @@ mod test {
                     .unwrap()
                 })
                 .await
-                .reconstruct()
-                .as_u128();
-            assert_eq!((x, y, result), (x, y, expected));
+                .reconstruct();
+            assert_eq!((x, y, result.as_u128()), (x, y, expected));
         });
     }
 }
```

Comment:
I see a test for `integer_add::<_, BA64, BA32>` but not `integer_add::<_, BA32, BA64>`.

Reply:
Added a test for this! Luckily it passes =).